openmc/deplete/chain.py

```diff
@@ -104,6 +104,19 @@ def replace_missing(product, decay_data):
     return product
 
 
+_SECONDARY_PARTICLES = {
+    "(n,p)": ["H1"], "(n,d)": ["H2"], "(n,t)": ["H3"], "(n,3He)": ["He3"],
+    "(n,a)": ["He4"], "(n,2nd)": ["H2"], "(n,na)": ["He4"], "(n,3na)": ["He4"],
+    "(n,n3a)": ["He4"] * 3, "(n,2na)": ["He4"], "(n,np)": ["H1"],
+    "(n,n2a)": ["He4"] * 2, "(n,2n2a)": ["He4"] * 2, "(n,nd)": ["H2"],
+    "(n,nt)": ["H3"], "(n,nHe-3)": ["He3"], "(n,nd2a)": ["H2", "He4"],
+    "(n,nt2a)": ["H3", "He4", "He4"], "(n,2np)": ["H1"], "(n,3np)": ["H1"],
+    "(n,n2p)": ["H1"] * 2, "(n,2a)": ["He4"] * 2, "(n,3a)": ["He4"] * 3,
+    "(n,2p)": ["H1"] * 2, "(n,pa)": ["H1", "He4"],
+    "(n,t2a)": ["H3", "He4", "He4"], "(n,d2a)": ["H2", "He4", "He4"],
+    "(n,pd)": ["H1", "H2"], "(n,pt)": ["H1", "H3"], "(n,da)": ["H2", "He4"]}
```

Comment:
So it looks like the way these are used, we're actually only checking membership, i.e. we don't care how many of each particle is produced. What do you think about just making the values sets rather than lists?

Reply:
I considered that and can easily make that switch. I went with lists over sets because, in the future, we may want to know how many secondary particles are produced to update the depletion chain. This could allow us to add as `(n,3a)` reaction to the depletion chain producing `He4` with a branching ratio of 3. Could be useful for #1321

Reply:
Good idea -- let's leave it as is